views/partials/header.ejs

```diff
@@ -514,6 +514,9 @@
 							<div>
 								<label><input type="checkbox" name="option_display_compact_view" id="option_display_compact_view"> Compact view</label>
 							</div>
+                                                        <div>
+ 								<label><input type="checkbox" name="option_proposed_team_icon" id="option_proposed_team_icon">Use shields instead of guns</label>
```

Comment:
Need to also change this bit, option_display_proposed...

Reply:
Accepted. Will be resolved in the next commit